src/lib/constants.ts

```diff
@@ -18,7 +18,7 @@ declare global {
 
 export const AM_COOKIE_NAME = import.meta.env.VITE_FR_AM_COOKIE_NAME;
 export const AM_DOMAIN_PATH = import.meta.env.VITE_FR_AM_URL;
-export const AM_DOMAIN = extractDomainFromUrl(import.meta.env.VITE_FR_AM_URL);
+export const AM_DOMAIN = extractDomainFromUrl(import.meta.env.VITE_FR_AM_URL || "loginwidget.example.com");
```

Comment:
Can we add a TODO comment here to re-evaluate this as it feels a bit like a hack without understanding the underlying issue here?